dev/benchmarks/microbenchmarks/lib/foundation/clamp.dart

```diff
@@ -0,0 +1,52 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart' show clampDouble;
+
+import '../common.dart';
+
+const int _kNumIterations = 1000000;
+
+void main() {
+  assert(false,
+      "Don't run benchmarks in checked mode! Use 'flutter run --release'.");
```

Comment:
nit: jonah fixed the checked langauge elsewhere in a recent page, might as well keep it consistent here.

Reply:
I don't understand, please clarify.  This is what we have in front of the other benchmarks.

Reply:
See e.g. https://github.com/flutter/flutter/blob/master/dev/benchmarks/microbenchmarks/lib/gestures/gesture_detector_bench.dart#L14

You may be synced to a commit before Jonah made the change.

Reply:
checked mode is from dart 1

Reply:
to enable asserts, you used to run `dart --checked` or `flutter run --checked`, but that hasn't existed for ~5 years

Reply:
Got it, thanks, fixed.